pkg/skr/common/defaultiprange/waitIpRangeReady.go

```diff
@@ -0,0 +1,30 @@
+package defaultiprange
+
+import (
+	"context"
+	"fmt"
+	cloudresourcesv1beta1 "github.com/kyma-project/cloud-manager/api/cloud-resources/v1beta1"
+	"github.com/kyma-project/cloud-manager/pkg/composed"
+	"github.com/kyma-project/cloud-manager/pkg/util"
+	"k8s.io/apimachinery/pkg/api/meta"
+)
+
+func waitIpRangeReady(ctx context.Context, st composed.State) (error, context.Context) {
+	state := st.(State)
+
+	if composed.MarkedForDeletionPredicate(ctx, state) {
+		return nil, nil
+	}
+
+	isReady := meta.IsStatusConditionTrue(state.GetSkrIpRange().Status.Conditions, cloudresourcesv1beta1.ConditionTypeReady)
```

Comment:
Would it make sense to implement IsReady on SKR IpRange, so this becomes `state.GetSkrIpRange().IsReady()`?